api/src/pcapi/static/backofficev3/js/addons/pc-form-field.js

```diff
@@ -5,14 +5,95 @@
  *
  */
 class PcFormField extends PcAddOn {
+  static ID = 'PCFormFieldId'
+  static PC_FORM_FIELD_FIELD_CLASS = 'pc-form-field-field'
+  static PC_FORM_FIELD_FIELD_EMPTY_SELECTOR = '.pc-form-field-empty-container'
+  static PC_REQUIRED_SELECTOR = '.pc-required'
 
   initialize = () => {
-    document.querySelectorAll(".pc-form-field-empty-container").forEach(($emptyFormField) => {
-      $emptyFormField.querySelectorAll(".value-element-form").forEach(($valueElementForm) => {
+    this.formFieldEmpty.forEach(($emptyFormField) => {
+      $emptyFormField.querySelectorAll(this.constructor.PC_REQUIRED_SELECTOR).forEach(($requiredField) => {
+        $requiredField.required = false
+      })
+      $emptyFormField.querySelectorAll(this.app.addons.PCFieldListId.constructor.FIELD_ELEMENT_BEARING_VALUE_SELECTOR).forEach(($valueElementForm) => {
         $valueElementForm.removeAttribute("name")
         $valueElementForm.removeAttribute("id")
       })
-    });
+    })
+    this._hideUnselectedFieldsAll()
+  }
+
+  get formFieldEmpty() {
+    return document.querySelectorAll(this.constructor.PC_FORM_FIELD_FIELD_EMPTY_SELECTOR)
+  }
+
+  bindEvents = () => {
+    this.initialize()
+    EventHandler.on(document.body, 'change', `.${this.constructor.PC_FORM_FIELD_FIELD_CLASS} > .pc-select-with-placeholder`, this.#redrawFields)
+  }
+  unbindEvents = () => {
+    EventHandler.off(document.body, 'change', `.${this.constructor.PC_FORM_FIELD_FIELD_CLASS} > .pc-select-with-placeholder`, this.#redrawFields)
+  }
+
+  #redrawFields = ($event) => {
+    const $formField = event.target.closest(`.${this.constructor.PC_FORM_FIELD_FIELD_CLASS}`)
```

Comment:
On accéde aux static via la class name, pourquoi le faire via `this.constructor` maintenant ?